src/routes/auth.js

```diff
@@ -0,0 +1,12 @@
+const express = require('express');
+const authController = require('../controllers/auth');
+
+const router = express.Router();
+
+router.post('/signin', authController.signin);
+
+router.post('/signup', authController.signup);
+
+router.post('/google', authController.signinWithGmail);
```

Comment:
you can do /signin/google as an endpoint for google sing in instead of just google